x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/components/alert_summary_widget_full_size.tsx

```diff
@@ -0,0 +1,161 @@
+import { Axis, Chart, CurveType, LineSeries, Position, ScaleType, Settings } from '@elastic/charts';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer, EuiText, useEuiTheme } from '@elastic/eui';
+import {
+  ACTIVE_ALERT_LABEL,
+  ACTIVE_COLOR,
+  ALL_ALERT_LABEL,
+  RECOVERED_ALERT_LABEL,
+  RECOVERED_COLOR,
+  TOOLTIP_DATE_FORMAT,
+} from './constants';
+import { Alert } from '../../../../../hooks/use_load_alert_summary';
+
+export interface AlertsSummaryWidgetFullSizeProps {
+  activeAlertCount: number;
+  activeAlerts: Alert[];
+  recoveredAlertCount: number;
+  recoveredAlerts: Alert[];
+}
+
+export const AlertsSummaryWidgetFullSize = ({
+  activeAlertCount,
+  activeAlerts,
+  recoveredAlertCount,
+  recoveredAlerts,
+}: AlertsSummaryWidgetFullSizeProps) => {
+  const isDarkMode = useUiSetting<boolean>('theme:darkMode');
+  const { euiTheme } = useEuiTheme();
+  const chartTheme = [
+    EUI_SPARKLINE_THEME_PARTIAL,
+    {
+      ...(isDarkMode ? EUI_CHARTS_THEME_DARK.theme : EUI_CHARTS_THEME_LIGHT.theme),
+      chartMargins: {
+        left: 10,
+        right: 10,
+        top: 10,
+        bottom: 10,
+      },
+    },
+  ];
+
+  return (
+    <EuiPanel
+      element="div"
+      data-test-subj="alertSummaryWidget"
+      hasShadow={false}
+      paddingSize="none"
+    >
+      <EuiFlexGroup direction="row">
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem>
+              <EuiFlexGroup gutterSize="xl" alignItems="flexStart" responsive={false}>
+                <EuiFlexItem>
+                  <EuiText color={euiTheme.colors.text}>
+                    <h3 data-test-subj="totalAlertsCount">
+                      {activeAlertCount + recoveredAlertCount}
+                    </h3>
+                  </EuiText>
+                  <EuiText size="xs" color="subdued">
+                    {ALL_ALERT_LABEL}
+                  </EuiText>
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiText color={euiTheme.colors.dangerText}>
+                    <h3 data-test-subj="activeAlertsCount">{activeAlertCount}</h3>
+                  </EuiText>
+                  <EuiText size="xs" color="subdued">
+                    {ACTIVE_ALERT_LABEL}
+                  </EuiText>
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiText color={euiTheme.colors.successText}>
+                      <h3 data-test-subj="recoveredAlertsCount">{recoveredAlertCount}</h3>
+                    </EuiText>
+                  </EuiFlexItem>
+                  <EuiText size="xs" color="subdued">
+                    {RECOVERED_ALERT_LABEL}
+                  </EuiText>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="l" />
+      <Chart size={['100%', 170]}>
```

Comment:
nit -> can we create a constant?

Reply:
Since performance-wise it does not have an impact, I will check where it helps readability and I will update them.